src/components/StudioCard/StudioCard.js

```diff
@@ -1,7 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
+import {
+  Card,
+  ImgBox,
+  Content,
+  ContentPrice,
+  Title,
+  StyledSlider,
+  StyledLink,
+  WishBtn,
+  OutlineHeart,
+  FillHeart,
+} from './StyleStudioCard';
 
-const StudioCard = () => {
-  return <div>StudioCard</div>;
+const StudioCard = ({ list, settings }) => {
+  const [isWishlistAdd, setIsWishlistAdd] = useState(false);
+
+  const handleWishBtn = () => {
+    setIsWishlistAdd(!isWishlistAdd);
+  };
+
+  return (
+    <Card>
```

Comment:
```suggestion
    <S.Card>
```
위에 드린 리뷰와 이어서 말씀드리면 이렇게 적용할 수 있겠습니다